fastlane/Fastfile

```diff
@@ -30,27 +30,29 @@ end
 
 desc "Release a new version"
 lane :release do |options|
```

Comment:
This is actually "starting" a release. Should we reflect that in the description and on the name?